obsplus/events/json.py

```diff
@@ -6,136 +6,44 @@
 from typing import Union
 
 import obspy
-from obspy.core.event import ResourceIdentifier, QuantityError
+from obspy.core.event import Event, Catalog
 
-from obsplus.utils.events import obj_to_dict, make_class_map
-from obsplus.constants import JSON_KEYS_TO_POP
+import obsplus.events.schema as event_schema
 
 JSON_SERIALIZER_VERSION = "0.0.0"  # increment when serialization changes
 
 
 # -------------------- events to json functions
 
 
-def cat_to_json(catalog: obspy.Catalog) -> str:
+def cat_to_json(events: Union[Catalog, Event]) -> str:
```

Comment:
Good to see single events explicitly allowed, but would it be possible to keep the variable as `catalog`? I'm pretty sure I use this function directly in a few of my codes, and I'm concerned that I may have used a kwarg at some point.